docs/Developers/Git_Basics.rst

```diff
@@ -70,7 +70,12 @@ Creating a new branch
 ---------------------
 
 To start working on an issue, you first need to create a new branch where you
-will work.
+will work. The default branch is "master" but when you want to add a new
+feature or fix a bug, no matter how big or how small, you spawn a new branch to
+encapsulate your changes. This makes sure that unstable code is never
```

Comment:
The reason to never use "master" is that a forks' "master" should **always** follow (be syncronised with) the main "master", and that can not happen if it has new commits.